tests/components/onewire/__init__.py

```diff
@@ -178,5 +178,9 @@ def _setup_owproxy_mock_device_reads(
 
     # Setup sub-device reads
     device_sensors = mock_device.get(platform, [])
+    if platform is Platform.SENSOR and device_id.startswith("12"):
```

Comment:
Should this be `12.` so you don't match 112 ?

Reply:
No need. The "device family" is always the first two characters.